tests/Unit/Domain/Services/TalkRating/YesNoRatingTest.php

```diff
@@ -36,6 +36,15 @@ public function testValidRatings($rating, $valid)
         $this->assertSame($valid, $yesno->isValidRating($rating));
     }
 
+    public function testGetRatingNameReturnsYesNo()
+    {
+        $mockAuth = Mockery::mock(Authentication::class);
+        $mockAuth->shouldReceive('userId');
+        $metaMock = Mockery::mock(TalkMeta::class);
+        $yesno    = new YesNoRating($metaMock, $mockAuth);
+        $this->assertSame('yesno', $yesno->getRatingName());
```

Comment:
Just as an idea, what do you think about giving a bit more space for breathing between lines of code?

I usually like to at least add an empty line between each of the *arrange*, *act*, and *assert* phases in a test.